src/pages/ProductDetail/ProductDetail.js

```diff
@@ -0,0 +1,138 @@
+import { useNavigate } from 'react-router-dom';
+import React, { useState, useEffect } from 'react';
+import './ProductDetail.scss';
+import Count from '../../components/Count/Count';
+
+const ProductDetail = () => {
+  const navigate = useNavigate();
+  const [countNumber, setCount] = useState(1);
+  const [product, setProduct] = useState([]);
+
+  useEffect(() => {
+    fetch('./data/product.json')
+      .then(res => res.json())
+      .then(data => setProduct(data.product));
+  }, []);
+  return (
+    <div className="productDetail">
+      <div className="productBox">
+        <div className="itemImageBox">
+          <img
+            src="./images/6.jpg"
+            alt="productImage"
+            className="productImage"
+          />
+        </div>
+        {product.map(product => (
+          <div className="productDescription" key={product.id}>
+            <h1 className="productName">상품이름:{product.name}</h1>
+            <p className="mainDescription">{product.description}</p>
+            <ul className="descriptionBox">
+              <li className="usingType">TYPE</li>
+              <p className="itemUsingType">{product.type}</p>
+              <li className="usingType">용도</li>
+              <p className="itemUsingType">{product.using}</p>
+              <li className="usingType">SIZE</li>
+              <p className="itemUsingType">{product.size}</p>
+              <li className="usingType">THICKNESS</li>
+              <p className="itemUsingType">{product.thickness}</p>
+              <li className="usingType">WEIGHT</li>
+              <p className="itemUsingType">{product.weight}</p>
```

Comment:
ul 태그 하위에는 li태그로 감싼 자식요소가 들어오는것이 일반적입니다. 
따라서
```suggestion
              <li className="usingType">
                  <div> TYPE </div>
                  <p className="itemUsingType">{product.type}</p>
              </li>
              
```
위와 같은 형태로 만들어 주시는게  더욱 sementic한 마크업 방식이며, UI를 구성할 때에도 편리 할 것 같습니다!

Reply:
수정하였습니다.
의미없는 div 태그를  남용하면 안된다고 생각했는데, 너무 단편적으로 봤네요 .
주말동안 시멘틱한 마크업 방식에대해 공부해 보겠습니다.